openedx/features/content_type_gating/partitions.py

```diff
@@ -93,8 +100,42 @@ def access_denied_message(self, block, user, user_group, allowed_groups):
         return None
 
     def _is_audit_enrollment(self, user, block):
+        """
+        Returns True if viewing course as:
+            1 - `Learner in Audit`
```

Comment:
Doc string should not explain steps of what code does, the code should be readable itself. 
Make sure this doc string explains an abstract of what this method does.